go/chat/storage/storage.go

```diff
@@ -187,6 +187,24 @@ func (s *Storage) Merge(ctx context.Context, convID chat1.ConversationID, uid gr
 	return nil
 }
 
+// getSupersedes must be called with a valid msg
+func (s *Storage) getSupersedes(msg chat1.MessageUnboxed) (chat1.MessageID, libkb.ChatStorageError) {
```

Comment:
Could we add a comment about how there's a Supersedes field in the ClientHeader, but we're choosing to ignore it in favor of verifiable data?

Also, my upcoming PR is going to turn the `Delete` field into a `Deletes`. Would it be ok to make this function return a list from the start?